a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -232,6 +232,10 @@ def get_new_lb_networking_subflow(self, topology):
             vthunder_tasks.VThunderComputeConnectivityWait(
                 name=a10constants.MASTER_CONNECTIVITY_WAIT,
                 requires=(a10constants.VTHUNDER, constants.AMPHORA)))
+        if topology == constants.TOPOLOGY_SINGLE:
```

Comment:
This flow isn't getting used by rack flow. 
However I inserted this here for single topology flow, which I assumed as part of the story. 

Referring to your  side note: 
Without that topology check, it was throwing error in further task for configuring vrrp saying, the url doesn't exist. Hence added it to avoid it entirely for ACTIVE/STANDBY mode.

Anyways, I think I can remove this task from here, as its not needed.

Reply:
Strange. Our base case is Standalone and then we handle all the backup stuff if topology is Active/Standby.

We can investigate further once this story comes back around again for  non-hardware.

Reply:
ok